x-pack/legacy/plugins/siem/public/components/timeline/body/column_headers/column_header.tsx

```diff
@@ -34,69 +32,83 @@ const ColumnHeaderComponent: React.FC<ColumneHeaderProps> = ({
   draggableIndex,
   header,
   timelineId,
+  isDragging,
   onColumnRemoved,
   onColumnResized,
   onColumnSorted,
   onFilterChange,
   sort,
 }) => {
-  const [isDragging, setIsDragging] = React.useState(false);
-  const handleResizeStop: ResizeCallback = (e, direction, ref, delta) => {
-    onColumnResized({ columnId: header.id, delta: delta.width });
-  };
+  const resizableEnable = useMemo(() => ({ right: true }), []);
```

Comment:
This pure function doesn't appear to capture any variables via closure, (as opposed to  for example, `resizableSize` below it).

Consider declaring `resizableEnable` outside the scope of this component, because it doesn't appear that every instance of the component needs its own memoized version of `resizableEnable`.